python/py-crank/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-crank
+version             0.8.1
+revision            0
+
+platforms           darwin
+supported_archs     noarch
+license             MIT
+maintainers         nomaintainer
+
+description         Generalized Object based Dispatch mechanism for use across frameworks
+long_description    {*}${description}.
+
+homepage            https://github.com/TurboGears/crank
+
+checksums           rmd160  9b3a788eadcfdf64515f5651e3be67e191f09fec \
+                    sha256  c22b94ab6e9d27be47733002bd143b3b79d1e8c030cba0b18691cbc1b5907d7f \
+                    size    10655
+
+python.versions     38
+
+if {${name} ne ${subport}} {
+    depends_build-append    port:py${python.version}-setuptools
+
+    livecheck.type      none
+} else {
+    livecheck.type  pypi
```

Comment:
Minor nitpick. Please align the spacing for both live check lines. Move "none" back 4 spaces.

Reply:
Yes, I saw that after the push, but I was thinking the other way. I usually go with:

```
    livecheck.type      none
} else {
    livecheck.type      pypi
}
```

I'd prefer that, if it's ok. (to match the tabbbing of those outside the if)

Reply:
Done √

Reply:
Either way is fine.